packages/gasket-helper-intl/lib/index.d.ts

```diff
@@ -33,7 +35,17 @@ export type LocalePathPart = string;
 /**
  * Callback which receives a context object for resolving a LocalePathPath
  */
-export type LocalePathThunk = (context: {}) => LocalePathPart;
+export type LocalePathThunk = (context: {
+    // Any server render
+    req?: IncomingMessage,
+    res?: OutgoingMessage
+
+    // Possible Next.js contexts
+    // @see: https://nextjs.org/docs/api-reference/data-fetching/get-static-props#context-parameter
```

Comment:
Could the properties from those docs be expanded into this type? Does next.js maybe expose a context type that can be imported?